### PR TITLE
Add IdP secrets to arm01 job envs

### PR DESCRIPTION
**Pull request by LorbusChris**
/assign @alvaroaleman

### CODE REVIEW

File: ci-operator/jobs/openshift/release/openshift-release-master-postsubmits.yaml [hunk 1]
```diff
@@ -336,6 +336,17 @@ postsubmits:
...
+        env:
+        - name: arm01_id
+          valueFrom:
+            secretKeyRef:
+              key: arm01-id
+              name: arm01-dex-oidc
+        - name: github_client_id
+          valueFrom:
+            secretKeyRef:
+              key: arm01_github_client_id
```
Comment (alvaroaleman) [inline ci-operator/jobs/openshift/release/openshift-release-master-postsubmits.yaml:348]: This doesn't exist

File: ci-operator/jobs/openshift/release/openshift-release-master-presubmits.yaml [hunk 2]
```diff
@@ -95,6 +95,16 @@ presubmits:
...
           value: /tmp
+        - name: arm01_id
+          valueFrom:
+            secretKeyRef:
+              key: arm01-id
+              name: arm01-dex-oidc
+        - name: github_client_id
+          valueFrom:
+            secretKeyRef:
+              key: arm01_github_client_id
```
Comment (alvaroaleman) [inline ci-operator/jobs/openshift/release/openshift-release-master-presubmits.yaml:106]: ditto

**Comment by LorbusChris** [comment]
@alvaroaleman oh right. From https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/18335/pull-ci-openshift-release-master-arm01-dry/1389964653412487168#1:build-log.txt%3A7 it looks like it probably should?
```
Required value: template.parameters[0]: parameter github_client_id is required and must be specified
```
Can you create it for me? Or give me a pointer, if I can do it myself?

**Comment by LorbusChris** [comment]
Eek. Looks like I missed that one in https://github.com/openshift/release/blob/master/core-services/ci-secret-bootstrap/_config.yaml

**Comment by alvaroaleman** [comment]
@LorbusChris can we just leave that out for now? It won't be needed if you dont set up github authentication

**Comment by LorbusChris** [comment]
works for me, too. Will update.

**Review by alvaroaleman — APPROVED** [review]

**Pull request merged**